codeHF/dpl-config_run3.json

```diff
@@ -2937,6 +2940,7 @@
         "etaMax": "0.8"
     },
     "track-propagation": {
+        "minPropagationDistance": "83.0999985",
```

Comment:
The trailing digits do not seem right.

Reply:
OK.
On hyperloop, the default is 5.
In code, the default (https://github.com/AliceO2Group/O2Physics/blob/master/Common/TableProducer/trackPropagation.cxx#L72C91-L72C103) is 83.0 + 0.1.
Which one should I put here?

Reply:
Hyperloop value.